src/screens/contacts/contacts.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { View, Text, SafeAreaView, Button } from 'react-native';
+import { ListItem, Avatar, Overlay, Input } from 'react-native-elements';
+import { FlatList, ScrollView, TouchableOpacity } from 'react-native-gesture-handler';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import styles from './contacts.styles';
+
+import { _getUser } from '../../utils/asyncUtil';
+
+class ContactsScreen extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            allUsers: [],
+           
+            listUsers: []
+        }
+    }
+
+    async componentDidMount() {
+        let user = await _getUser()
+        this.convertUserObjectToArray(this.props.usersProfile, user);
+    }
+
+    convertUserObjectToArray = (usersObj = {}, own) => {
+        let usersArr = [];
+        Object.keys(usersObj).forEach((user = {}) => {
+            if (own !== user) {
+                usersArr.push({ ...usersObj[user], key: user });
+            }
+        });
+        this.setState({ allUsers: usersArr, listUsers: usersArr, mainUser: own });
+    }
+
+    listItem = ({ item }) => {
```

Comment:
we can make this function as a separate functional component instead of a normal function inside this "ContactsScreen" component.